rxandroidble/src/main/java/com/polidea/rxandroidble/internal/operations/RxBleRadioOperationScan.java

```diff
@@ -46,8 +46,11 @@ public void run() {
     }
 
     public void stop() {
+
         // TODO: [PU] 29.01.2016 https://code.google.com/p/android/issues/detail?id=160503
-        rxBleAdapterWrapper.stopLeScan(leScanCallback);
+        if (startLeScanStatus) {
```

Comment:
Good point. What do you think about making status `volatile` instead of using a monitor?